src/patternfly/components/Accordion/docs/code.md

```diff
@@ -15,11 +21,11 @@
 
 | Class | Applied to | Outcome |
 | -- | -- | -- |
-| `.pf-c-accordion` | `<dl>` | Initiates an accordion component. **Required**|
-| `.pf-c-accordion__toggle` | `<dt><h3><button>` | Initiates a toggle in the accordion. **Required** |
+| `.pf-c-accordion` | `<div>`, `<dl>` | Initiates an accordion component. **Required**|
+| `.pf-c-accordion__toggle` | `<h3><button>`, `<dt><button>` | Initiates a toggle in the accordion. **Required** |
```

Comment:
Since this is the only `<button>` in the accordion, maybe we can change this:
`<h3><button>`, `<dt><button>`
to just be this:
`<button>`

One benefit of this is not having to explain that the `<h3>` should really be a heading level that fits the rest of the heading outline on your page.

Reply:
per our offline conversation, updating this to be <h1 - h6>